packages/odie-client/src/components/message/messages-container.tsx

```diff
@@ -47,24 +47,28 @@ export const MessagesContainer = ( { currentUser }: ChatMessagesProps ) => {
 		( chat?.status === 'loaded' || chat?.status === 'closed' ) && setChatLoaded( true );
 	}, [ chat ] );
 
-	const shouldLoadChat: boolean =
-		! shouldUseHelpCenterExperience || ( shouldUseHelpCenterExperience && chatMessagesLoaded );
+	const shouldLoadChat: boolean = chatMessagesLoaded;
 
 	// Used to apply the correct styling on messages
 	const isNextMessageFromSameSender = ( currentMessage: string, nextMessage: string ) => {
 		return currentMessage === nextMessage;
 	};
 
+	const statusArray =
+		experimentName === 'give_a_chance'
+			? [ 'sending', 'transfer' ]
+			: [ 'sending', 'dislike', 'transfer' ];
```

Comment:
Please change the experiment name. I also don't understand what's happenning here.